### ISSUE TITLE
"bacon test -- -p some_package" doesn't work

### ISSUE THREAD

**Issue by Hubro** [Canop/bacon#134]
From [the documentation](https://dystroy.org/bacon/cookbook/#variable-arguments), it sounds like it should be possible to pass flags to `bacon test`.

For example, this works perfectly:

```
cargo test -p my_sub_package
```

But this doesn't work, instead it tries to compile my entire workspace, which crashes:

```
bacon test -- -p my_sub_package
```

**Comment by Canop** [comment]
It works for me.

Can you please give your configuration and/or build a complete reproductible example ?

**Comment by Hubro** [comment]
> It works for me.
> 
> Can you please give your configuration and/or build a complete reproductible example ?

Sure, here's a minimal example:

https://github.com/Hubro/bacon-issue-134-demo

To reproduce:

1. `cargo test`

   Fails, because there's a syntax error in crate `dos`
2. `cargo test -p uno`

   Succeeds, because only crate `uno` is built
3. `bacon test -- -p uno`

   Fails with syntax error from crate `dos`, just like the command `cargo test`. It's as if the option `-p uno` wasn't forwarded at all.

```
$ bacon --version
bacon 2.8.1
```

**Comment by that-ambuj** [comment]
Hi, I also had this issue when specifying `--bin` option for test. I have an alternate solution that does not require `bacon.toml` file. The solution is to use an extra `--` when specifying options to `cargo` instead of `bacon`.

In this case @Hubro  would have change only one thing about the command, This would also keep the fix provided by #124 intact.

```bash
bacon test -- -- -p my_sub_package
```

**Comment by Canop** [comment]
Right now I'm tempted to solve the problem with just some comment in the bacon.toml file, and maybe the website

**Comment by wmmc88** [comment]
> Hi, I also had this issue when specifying `--bin` option for test. I have an alternate solution that does not require `bacon.toml` file. The solution is to use an extra `--` when specifying options to `cargo` instead of `bacon`.
> 
> In this case @Hubro would have change only one thing about the command, This would also keep the fix provided by #124 intact.
> 
> ```shell
> bacon test -- -- -p my_sub_package
> ```

I think this work around may not work anymore. tested on `bacon 2.13.0`. had to `bacon --init` and remove the ` "--", "--color", "always", # see https://github.com/Canop/bacon/issues/124` line
